File: algorithm/baekjoon/14JUN21/8_2941.py
#ljes=njak

#변경할 알파벳
chg = ['c=','c-','dz=','d-','lj','nj','s=','z=']
word = input()  #ljes=njak

#chg 요소를 반복하여 word에 있는 요소를 *로 변경
for i in chg:
    word = word.replace(i,'.')
#'.e..ak'
#len은 총 6
print(len(word))

File: algorithm/baekjoon/15JUN21/8_2941.js
// let input = 'ljes=njak'
const input = require('fs').readFileSync('dev/stdin').toString().trim()
let word = input
let croatia = [/c=/g, /c-/g,/dz=/g,/d-/g,/lj/g,/nj/g,/s=/g,/z=/g];
// g flag(/regex/g)
// : 표현식을 만족시키는 패턴이 한 개 이상 있는지 검사합니다. match() 함수의 리턴 값이 스트링 그룹, 즉 배열(Array) 형태입니다.

//map : 새로운 배열로 만들어주기 위한 함수
croatia.map(Element => word = word.replace(Element,'.'));
console.log(word.length)